ocaml/ocaml-findlib/Portfile

```diff
@@ -30,25 +34,18 @@ long_description    The findlib library provides a scheme to manage reusable \
 homepage            http://projects.camlcity.org/projects/findlib.html
 master_sites        http://download.camlcity.org/download/
```

Comment:
We should probably use Github as the master site at this point for downloads, given that camlcity is essentially dead. Yes, I know the project page on github points to camlcity.